src/utils/media-utils.js

```diff
@@ -1,8 +1,9 @@
 const whitelistedHosts = [/^.*\.reticulum\.io$/, /^.*hubs\.mozilla\.com$/, /^hubs\.local$/];
 const isHostWhitelisted = hostname => !!whitelistedHosts.filter(r => r.test(hostname)).length;
 let mediaAPIEndpoint = "/api/v1/media";
-if (process.env.NODE_ENV === "development") {
-  mediaAPIEndpoint = `https://${process.env.DEV_RETICULUM_SERVER}${mediaAPIEndpoint}`;
+
+if (process.env.RETICULUM_SERVER) {
+  mediaAPIEndpoint = `https://${process.env.RETICULUM_SERVER}${mediaAPIEndpoint}`;
```

Comment:
It seems like we expect `RETICULUM_SERVER` to always be set now -- can we simplify places like this to instead be e.g.
```Javascript
let mediaAPIEndpoint = "//${process.env.RETICULUM_SERVER}/api/v1/media"
```
?

Reply:
yeah I wasn't sure -- it may be useful to keep things so that if the environment variable isn't set it uses relative paths. (for example, perhaps there are situations where you don't know the domain name at build time?)

Reply:
eg i could imagine a model where there's a built version of Hubs that you can throw it behind two different reticulum servers and it works

Reply:
Hmm, no idea whether I think that is the case or not, but then at least I would rewrite it without a branch, e.g.
```
let apiHost = process.env.RETICULUM_SERVER || location.host;
let apiEndpoint = `${apiHost}/api/v1/media`
```
I don't care if you change it now though.

Reply:
Another random thought -- maybe it would be a nice time to move locations of our API endpoints and any associated utilities for making paths to them into a separate module, since there are a few of them.

Reply:
sounds like a good idea to me, i probably will leave that to a future PR though.